docs/content/en/docs/reference/vsphere/vsphere-ovas.md

```diff
@@ -100,75 +99,74 @@ environment variables to run `govc` GOVC_USERNAME, GOVC_PASSWORD and GOVC_URL.
     govc vm.markastemplate <name of new VM>
     ```
 
-
 ## Important Additional Steps to Tag the OVA
 
 ### Using vCenter UI
 
 #### Tag to indicate OS family
 
-1. Select the template that was newly created in the steps above and navigate to *Summary* -> *Tags*.
+1. Select the template that was newly created in the steps above, and navigate to *Summary* -> *Tags*.
    ![Import ova wizard](/images/ss10.jpg)
-1. Click *Assign* -> *Add Tag* to create a new tag and attach it
+1. Click *Assign* -> *Add Tag* to create a new tag and attach it.
    ![Import ova wizard](/images/ss11.jpg)
-1. Name the tag *os:ubuntu* or *os:bottlerocket*
+1. Name the tag *os:ubuntu* or *os:bottlerocket*.
    ![Import ova wizard](/images/ss12.jpg)
 
 #### Tag to indicate eksd release
 1. Select the template that was newly created in the steps above and navigate to *Summary* -> *Tags*.
    ![Import ova wizard](/images/ss10.jpg)
-1. Click *Assign* -> *Add Tag* to create a new tag and attach it
+1. Click *Assign* -> *Add Tag* to create a new tag and attach it.
    ![Import ova wizard](/images/ss11.jpg)
-1. Name the tag *eksdRelease:{eksd release for the selected ova}*, for example *eksdRelease:kubernetes-1-20-eks-5* for the 1.20 ova. You can find the rest of eksd releases in the previous [section](#import-an-ovaovf-template-to-vsphere). If it's the first time you add an `eksdRelease` tag, you would need to create the category first. Click on "Create New Category" and name it `eksdRelease`.
+1. Name the tag *eksdRelease:{EKS-D release for the selected ova}* (for example *eksdRelease:kubernetes-1-20-eks-5* for the 1.20 OVA). You can find the rest of the EKS-D releases in the previous [section](#import-an-ovaovf-template-to-vsphere). If it's the first time you add an `eksdRelease` tag, you would need to create the category first. Click on "Create New Category", and name it `eksdRelease`.
    ![Import ova wizard](/images/ss13.png)
 
 ### Using govc
 
 #### Tag to indicate OS family
 
-1. Create tag category
+1. Create tag category.
 
 ```
 govc tags.category.create -t VirtualMachine os
 ```
-1. Create tags os:ubuntu and os:bottlerocket
+1. Create tags *os:ubuntu* and *os:bottlerocket*.
```

Comment:
```suggestion
1. Create tags `os:ubuntu` and `os:bottlerocket`.
```
I think italics (*) should mostly be used for things that are substitutable, not literal.